.gitignore

```diff
@@ -9,3 +9,5 @@ __pycache__/
 
 /.clangd/
 /compile_commands.json
+
+.DS_store
```

Comment:
No explanatio given in the commit message. I don't know what this is. If this is a temporary directory of some application you're using, it should probably be better in your personal global gitignore.

Reply:
It's a file macOS creates when opening a directory in finder, I agree it should be in my personal gitignore.